addon.py

```diff
@@ -1777,6 +1777,18 @@ def list_collection(collection_id, mandatoryParams=None, parameter=None):
                                                 art=link_art,
                                                 folder_name=page_data['data']['attributes'].get('title'))
 
+        try:
+            if page_data['data']['meta']['itemsCurrentPage'] != page_data['data']['meta']['itemsTotalPages']:
+                nextPage = page_data['data']['meta']['itemsCurrentPage'] + 1
+                params = {
+                    'action': 'list_collection',
+                    'collection_id': collection_id,
+                    'page': nextPage
+                }
+                helper.add_item('Next Page', params, content='tvshows')
```

Comment:
Can you add translation to this?

Reply:
mandatoryParams and parameter is used when displaying videolists. Should those be added to this as well?

Reply:
fixed both things